DESCRIPTION

```diff
@@ -30,6 +30,7 @@ Authors@R: c(
   person("Oğuzhan", "Öğreden", role = c("aut"), comment = c(ORCID = "0000-0002-9949-3348")),
   person("Dalton", "Hance", role = c("aut"), email = "dhance@usgs.gov"),
   person("Daniel", "Nüst", role = c("aut", "cph"), email = "daniel.nuest@uni-muenster.de", comment = c(ORCID = "0000-0002-0024-5046"))
+  person("Petter", "Uvesten", role = c("aut", "cph"), email = "petter.uvesten.7362@student.lu.se")
```

Comment:
You need a comma at the end of the previous line.

Reply:
argh. Fixing.